client/client_test.go

```diff
@@ -224,6 +225,7 @@ var allTests = []func(t *testing.T, sb integration.Sandbox){
 }
 
 func TestIntegration(t *testing.T) {
+	t.Setenv("CONTAINERD_ENABLE_DEPRECATED_PULL_SCHEMA_1_IMAGE", "1")
```

Comment:
Huh, rootless still fails.

Reply:
Yeah, will have to try something else

Reply:
Okay, I know what's the problem with rootless tests. Look at this line: https://github.com/moby/buildkit/blob/master/util/testutil/workers/containerd.go#L190
I think it blocks propagation of variables defined in outside scope.

Reply:
Yep. Unless `sudo` is called with `-E`/`--preserve-env`, it uses a brand-new environment. Note that I am not sure it is a good idea to use these flags. Possibly we should just add `CONTAINERD_ENABLE_DEPRECATED_PULL_SCHEMA_1_IMAGE=1` to `c.ExtraEnv`.

Reply:
We can add it there if we want to support it, it should be removed soon entirely anyway

Reply:
Do we still need to support Schema 1 on our CI?

Reply:
That's a good question. There's only a single test that deliberately tests schema v1. But it's up to you, maintainers, to decide. As just a contributor, I am trying to go conservative route by preserving whatever functionality existed before my changes until explicit "yeah, burn it" is said :)

Reply:
I added a skip to see if it makes it go green. If it was up to me, I would vote to mark it as deprecated and remove the CI for it.